ArduCopter/crash_check.cpp

```diff
@@ -192,6 +192,11 @@ void Copter::parachute_check()
         return;
     }
 
+    if (parachute.release_initiated()) {
```

Comment:
why do we need that ? 
the only way to have _release_initiated = true; is to call release() that pass on parachute_release() that issue the disarm already. 
So I don't understant why we need to spam disarm command ?

Reply:
The issue we ran into was that the sink rate check is done from the parachute library itself ( https://github.com/tridge/ardupilot/blob/pr-copter-chute-flying/libraries/AP_Parachute/AP_Parachute.cpp#L128 ) and not here in `crash_check` so disarm was never commanded.

Looking again at `parachute_release()` we probably want to include the landing gear deployment here, too?

```
    // deploy landing gear
    landinggear.set_position(AP_LandingGear::LandingGear_Deploy);
```

Reply:
Ok I have look back a little more that won't be and issue to use this. It won't spam.

About the landinggear, that can be a touchy change as some will prefer to not try to deploy back the landing gear (possible interference with the parachute cable, you may touch the ground before deployement, etc etc.). Better make another PR to put the landing gear change. Doing Mount retract could be a good things on other matter !